### PR TITLE
Feature/display favorites

### PR DESCRIPTION
**Pull request by kammon10**
#### What's this PR do?
add display for favorites view and add/remove recipe card on favorite button click

#### Where should the reviewer start?
scripts.js

#### How should this be manually tested?
open whats cook'n and click on favorite button 

#### Any background context you want to provide?
none

#### What are the relevant tickets?
nope

#### Screenshots (if appropriate)
none

#### Questions:
nope

### CODE REVIEW

**Review by aliroemhildt — APPROVED** [review]
Nice work!

File: src/scripts.js [hunk 1]
```diff
@@ -21,24 +21,26 @@ let favoriteButtons = [];
...
 const filterTags = document.querySelectorAll('.tag');
+const favoritePageButton = document.getElementById('favoritesPage');
 
 
 
-// EVENT LISTENERS
-window.addEventListener('load', displayAllRecipes);
 
+// EVENT LISTENERS
+window.addEventListener('load', () => {
```
Comment (anewb87) [inline src/scripts.js:37]: Cleaning our functions up so we can use them over and over.

File: src/scripts.js [hunk 2]
```diff
@@ -47,17 +49,24 @@ function getRandomIndex(array) {
...
 
 function displayFavorites() {
-  hide([favoritePageButton, mainView, recipeSection]);
-  show([homeButton]);
-  //need other functions to be built out in order to finish this
+  hide([favoritePageButton]);
+  show([homeButton, filterSection, recipeSection, mainView]);
+  displayRecipes(currentUser.favorites);
+  recipeCards = document.querySelectorAll('.recipe-card-js');
+  recipeCards.forEach(card => {
```
Comment (anewb87) [inline src/scripts.js:64]: Glad we landed here after taking some time to realize we were over-complicating things.

File: src/scripts.js [hunk 3]
```diff
@@ -70,13 +79,13 @@ function filterAllRecipesByTag() {
...
-  recipeRepository.recipesToShow = recipeRepository.filterByTags(selectedTags);
-  displayRecipes();
+  displayRecipes(recipeRepository.filterByTags(selectedTags));
 }
 
-function displayRecipes() {
+function displayRecipes(recipes) {
   recipeSection.innerHTML = '';
-  recipeRepository.recipesToShow.forEach(recipe => {
+  console.log(recipes)
```
Comment (anewb87) [inline src/scripts.js:87]: We will take this out.

File: src/scripts.js [hunk 4]
```diff
@@ -100,6 +109,7 @@ function displayRecipes() {
 }
 
 function toggleFavoriteButton(e) {
+  console.log('e.target', e.target)
```
Comment (anewb87) [inline src/scripts.js:112]: Will remove as well.

**Review by anewb87 — APPROVED** [review]
We have a functioning favorites page!

**Review by cagallo — APPROVED** [review]
Looking great, almost done with functionality! Onto api calls!

**Pull request merged**